### ISSUE TITLE
Option to reset element position

### ISSUE THREAD

**Issue by ngaer** [PuruVJ/svelte-drag#13]
Hey @PuruVJ , thanks for the nice lib, it works great.

I got a use case that I can't achieve currently and I would like to discuss it:

I have an element that I should be able to drag in one state and shouldn't in another, this can be achieved using the `disabled` option. But when we are in a non-draggable state (`{ disabled: true }`), I also need the element to go back to its original position and this is something not easy to achieve. If I will remove the transform property from the element's style attribute, the element will get back to the original position. But when next time I will switch to `{ disabled: false }` and will start dragging, the element will jump back to the last position because the action has an internal state and use it to calculate new position on every move. 

So basically right now the only way to reset the position is to destroy the action and create it again. I think it would be nice to have a way to do that it using action's API. E.g.:
- Add another option (e.g. `resetPositionWhenDisabled`), if it's true then action will reset the position when it will get disabled.
- Add a way to set position through options, something like this:
```svelte
<script lang="ts">
  let offsetX = 0
  let offsetY = 0

  function reset() {
    offsetX = 0
    offsetY = 0
  }
</script>
<div
  use:draggable={{ offsetX, offsetY }}
  on:svelte-drag={{ detail } => { offsetX = detail.offsetX; offsetY = detail.offsetY }}
>
  Hello
</div>
```

What do you think?

**Comment by ngaer** [comment]
Great! I agree, the second option is more generic and might be useful for other use cases as well.

**Comment by PuruVJ** [comment]
Fixed in #20
